datastore/longhorn.go

```diff
@@ -867,6 +869,13 @@ func (s *DataStore) CreateDefaultNode(name string) (*longhorn.Node, error) {
 			return nil, err
 		}
 
+		// Attempt to create the specified Default Data Path on the disk, in case it doesn't exist.
+		nsPath := iscsi_util.GetHostNamespacePath(util.HostProcPath)
```

Comment:
Use `NamespaceExecutor`? https://github.com/longhorn/longhorn-manager/blob/master/util/util.go#L453

Reply:
Changing the code to make use of `NamespaceExecutor`.